src/commcare_cloud/fab/fabfile.py

```diff
@@ -383,33 +385,47 @@ def prepare_offline_deploy():
 
 
 @task
-def setup_release(keep_days=0):
+def setup_limited_release(keep_days=1):
+    setup_release(keep_days, full_cluster='False')
+
+
+@task
+def setup_release(keep_days=0, full_cluster='True'):
     """
     Setup a release in the releases directory with the most recent code.
     Useful for running management commands. These releases will automatically
     be cleaned up at the finish of each deploy. To ensure that a release will
     last past a deploy use the `keep_days` param.
 
     :param keep_days: The number of days to keep this release before it will be purged
+    :param keep_cluster: If True, only setup on webworkers[0] where the command will be run
 
     Example:
     fab <env> setup_release:keep_days=10  # Makes a new release that will last for 10 days
+    fab <env> setup_release:full_cluster=false  # Makes a new release solely on webworkers[0]
     """
     try:
         keep_days = int(keep_days)
     except ValueError:
         print(red("Unable to parse '{}' into an integer".format(keep_days)))
         exit()
 
+    try:
+        full_cluster = strtobool(full_cluster)
+    except ValueError:
+        print(red("Unable to parse '{}' into a boolean".format(full_cluster)))
+        exit()
+    if not full_cluster and not keep_days:
+        keep_days = 1
```

Comment:
If we go the route of the example code block in my comment above, then we won't need to have this other sort of secondary default here.